QUEST_LV_0100.tsv

```diff
@@ -6851,8 +6851,8 @@ QUEST_LV_0100_20150428_006850	The assistant commander Volda asked you to defeat
 QUEST_LV_0100_20150428_006851	Find the person who knows about the epitaph of Gailla Flurry and listen to his request.
 QUEST_LV_0100_20150428_006852	You found the luggage of the villager. Return the luggage to the villager and ask him about the contents in the epitaph.
 QUEST_LV_0100_20150428_006853	Defeat Blue Woodspirits that appeared near Obelisk
-QUEST_LV_0100_20150428_006854	$Goddess Saule lost the revelation from Bramble at the thorny forest of Kvailas Forest. Help the priests at Gate Route and Sirdgela Forest and look for the revelation at Kvailas Forest.
-QUEST_LV_0100_20150428_006855	Take the cable car from Srautas Canyon that will appear as you go down from the twin bridge of the crystal mine, and go up a bit to get to Gelly Plauteau.
+QUEST_LV_0100_20150428_006854	$The Goddess Saule lost the revelation from Brambles at the thorny forest of Kvailas forest. Help the priests at Gate Route and Sirdgela Forest and look for the revelation at Kvailas Forest.
```

Comment:
Brambles -> Bramble - Bramble is the boss monster.
Kvailas Forest is a map name though, so why the lowercase f?